### PR TITLE
feat(refDebounced): use MaybeComputedRef for ms

### PR DESCRIPTION
**Pull request by dominikklein**
### Additional context

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other (Type improvement)

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

File: packages/shared/refDebounced/index.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+import type { DebounceFilterOptions, MaybeComputedRef } from '../utils'
 
 /**
  * Debounce updates of a ref.
  *
  * @return A new debounced ref.
  */
-export function refDebounced<T>(value: Ref<T>, ms = 200, options: DebounceFilterOptions = {}): Readonly<Ref<T>> {
+export function refDebounced<T>(value: Ref<T>, ms: MaybeComputedRef<number> = 200, options: DebounceFilterOptions = {}): Readonly<Ref<T>> {
   if (ms <= 0)
```
Comment (wheatjs) [inline packages/shared/refDebounced/index.ts:12]: In this case, we should change this line to be `if(unref(ms) <= 0))` or just remove it all together. Currently if the initial value is `0`, you won't be able to change it later which is probably not the desired behavior.

Comment (dominikklein) [reply packages/shared/refDebounced/index.ts:12]: Ah, correct, I think when currently a `0` is used the complete usage of the `refDebounced` make no sense or?
So I think it makes sense to remove this check and support a reactive value. At least it's already working, only the type will not support it.

Comment (wheatjs) [reply packages/shared/refDebounced/index.ts:12]: Yep, so I suggest we just remove the check all together

**Comment by wheatjs** [comment]
Looks good to me, let's get a review from Anthony before merging

**Review by antfu — APPROVED** [review]

**Pull request merged**